tsconfig.json

```diff
@@ -47,6 +47,7 @@
   },
   "include": [
     "src",
+    "srv-src",
```

Comment:
Shouldn't this `tsconfig.json` only be for the frontend. Not sure...

Reply:
Not sure either... let's look into it together